internal/cloudapi/v2/compose.go

```diff
@@ -13,7 +13,12 @@ import (
 // GetBlueprintWithCustomizations returns a new Blueprint with all of the
 // customizations set from the ComposeRequest
 func (request *ComposeRequest) GetBlueprintWithCustomizations() (blueprint.Blueprint, error) {
-	var bp = blueprint.Blueprint{Name: "empty blueprint"}
+	var bp blueprint.Blueprint
+	if request.Blueprint != nil {
+		bp = request.Blueprint
```

Comment:
I know that this is just a PoC, but I wanted to point out that this commit doesn't compile. It confused me when reviewing this on a commit-by-commit basis.

Reply:
Yeah, wrong type of Blueprint :)